provisioning/etcd/values.yaml

```diff
@@ -0,0 +1,56 @@
+# DEFAULT VALUES: https://github.com/bitnami/charts/blob/master/bitnami/etcd/values.yaml
+image:
+  tag: 3.5.4-debian-11-r27
+
```

Comment:
Tieto hodnoty sa pouziju pre Helm, pri volani `generate.sh` (volal som to rucne, nie je to sucast pipeline).